scripts/generate-types.mjs

```diff
@@ -0,0 +1,33 @@
+import fs from 'node:fs/promises';
+
+const DATA_FILE = new URL('../globals.json', import.meta.url);
+
+const globals = JSON.parse(await fs.readFile(DATA_FILE));
+
+const getGroupTypeName = (group) => `Globals${group[0].toUpperCase() + group.slice(1).replaceAll('-', '')}`;
+
+const groups = {};
+
+console.log('// This file is autogenerated by scripts/generate-types.mjs');
+console.log('// Do NOT modify this file manually\n');
```

Comment:
It's cleaner to pushing to an array and then `console.log`'ing a `array.join('\n')` at the end.

Reply:
Fixed